mantidproject/mantid
# Workbench: User aimed documentation

**Description of work.**
This is the partially complete workbench documentation aimed at helping users get to grips with the new interface and operate with it.

**To test:**
- Build the user documentation target `docs-html`
- Then find the `/docs/html/index.html` file and then it should have a new section in reference documentation called `workbench` click this
- Here is the workbench documentation this file should be mostly complete
- Any of the files linked from this page are not finished.




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
